lib/core/src/display/shape/text/render.rs

```diff
@@ -0,0 +1,202 @@
+//! Rendering TextField.
+
+pub mod assignment;
+pub mod selection;
+
+use crate::prelude::*;
+
+use crate::display::object::DisplayObjectData;
+use crate::display::shape::glyph::font::FontRegistry;
+use crate::display::shape::glyph::font::FontRenderInfo;
+use crate::display::shape::glyph::system::GlyphSystem;
+use crate::display::shape::text::content::TextFieldContent;
+use crate::display::shape::text::content::TextFieldContentFullInfo;
+use crate::display::shape::text::cursor::Cursor;
+use crate::display::shape::text::cursor::Cursors;
+use crate::display::shape::text::render::assignment::GlyphLinesAssignment;
+use crate::display::shape::text::render::assignment::LineFragment;
+use crate::display::shape::text::render::selection::SelectionSpritesGenerator;
+use crate::display::shape::primitive::def::*;
+use crate::display::shape::primitive::system::ShapeSystem;
+use crate::display::symbol::geometry::compound::sprite::Sprite;
+
+use nalgebra::Vector2;
+use nalgebra::Vector3;
+use crate::display::shape::text::TextFieldProperties;
+use crate::display::world::World;
+
+
+// =======================
+// === RenderedContent ===
+// =======================
+
+type GlyphLine = crate::display::shape::glyph::system::Line;
```

Comment:
This is a line of TextField, right? If so, why its name is not just `Line` ? Being user of the lib, I would like to have `text_field::Line` type instead of `GlyphLine`, unless there are other lines than glyph lines.

Reply:
But this structure is not for users. The users see the `Line` struct which is part of `TextFieldContent`

Reply:
Added docs for this alias anyway.